main.go

```diff
@@ -69,19 +70,8 @@ type Config struct {
 	WorkDir       string `env:"working_dir,required"`
 }
 
-func main() {
-	var c Config
-	if err := stepconf.Parse(&c); err != nil {
-		failf("Failed to parse configs, error: %s", err)
-	}
-	stepconf.Print(c)
-
-	commits, err := releaseCommits(c.WorkDir, c.NewVersion)
-	if err != nil {
-		failf("Failed to get release commits, error: %v", err)
-	}
-
-	changelog, err := changelogContent(commits, c.NewVersion)
+func processChangelog(cp changelogPrinter, c Config) {
```

Comment:
it is a bit strange, that you pass Config as it is and pass some of its properties via changelog again